example/tsconfig.json

```diff
@@ -15,8 +18,21 @@
     "suppressImplicitAnyIndexErrors": true,
     "noUnusedLocals": true,
     "noUnusedParameters": true,
-    "allowSyntheticDefaultImports": true
+    "allowSyntheticDefaultImports": true,
+    "target": "es5",
+    "allowJs": true,
+    "skipLibCheck": true,
+    "strict": true,
+    "forceConsistentCasingInFileNames": true,
+    "resolveJsonModule": true,
+    "isolatedModules": true,
+    "noEmit": true
```

Comment:
if there is no good reason to i would suggest to keep the defaults as I am not sure of the side effects of changing the defaults as seen above

Reply:
@ttraenkler so I'm not entirely sure where these changes come from. Also, this is the example folder, which we're not using currently.